content/docs/command-reference/get.md

```diff
@@ -24,13 +24,13 @@ repository (e.g. source code, small image/other files). `dvc get` copies the
 target file or directory (found at `path` in `url`) to the current working
 directory. (Analogous to `wget`, but for repos.)
 
+> See `dvc list` for a way to browse repository contents to find files or
+> directories to download.
+
 > Note that unlike `dvc import`, this command does not track the downloaded
 > files (does not create a `.dvc` file). For that reason, it doesn't require an
 > existing DVC project to run in.
 
-> See `dvc list` for a way to browse repository contents to find files or
```

Comment:
not related to this PR?

Reply:
I did some general copy edits to the descriptions of all the commands related here (add, get, import, etc.) as I reviewed the -o/-to-remote stuff.

Reply:
it increases surfaces, please please let's try to avoid it as much as we can

Reply:
Sure, will extract. This is a common misunderstanding because I see these changes as related when I make them, as docs have lots of interconnections... Also I don't always have the capacity to remember small details for later and it's it's not always feasible to stash inner file chunks either. So it's tricky, sorry if this happens now and then 😢

Reply:
Ok, rolled back.